src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/pages/AddPreviousName.vue

```diff
@@ -1,7 +1,156 @@
-<template><div>ADD PREVIOUS NAME</div></template>
+<template>
+  <PageContainer>
+    <v-row>
+      <v-col cols="12">
+        <v-breadcrumbs class="pa-0" :items="items" color="primary">
+          <template #divider>/</template>
+        </v-breadcrumbs>
```

Comment:
I added a common Breadcrumb.vue component. You just add in the list of items that you want :)

Reply:
Got that with merge too, will do that now